barretenberg/cpp/src/barretenberg/polynomials/pow.hpp

```diff
@@ -63,6 +64,21 @@ template <typename FF> struct PowPolynomial {
      */
     FF univariate_eval(FF challenge) const { return (FF(1) + (challenge * (betas[current_element_idx] - FF(1)))); };
 
+    /**
+     * @brief Evaluate  \f$ ((1−X_{i}) + X_{i}\cdot \beta_{i})\f$ at the challenge point \f$ X_{i}=u_{i} \f$.
```

Comment:
leave TODO here to fix it?

Reply:
I'm not sure there's anything to fix. We do need to compute 28 betas so that the hashing constraints are independent of size (I left an issue in the relevant places for that) but we won't want to use those betas in this calculation.